test/api/test_ut_run.pkb

```diff
@@ -202,6 +202,130 @@ end;]';
   begin
     execute immediate 'drop package invalid_pckag_that_revalidates';
     execute immediate 'drop package parent_specs';
+  end;  
+
```

Comment:
Can you cover all 3 changed entry points (procedures) with tests so we have regression for each of them?